src/custom-element.js

```diff
@@ -1176,6 +1176,14 @@ function createBaseCustomElementClass(win) {
     }
 
     /**
+     * Returns reference to implementation after it has been built.
+     * @return {!Promise<?./base-element.BaseElement>}
```

Comment:
It'd be `!Promise<!./base-element.BaseElement>` - promise response is non-nullable.